### ISSUE TITLE
Nested progress bars not working in ios device

### ISSUE THREAD

**Issue by JosnaPriyaPereira** [crisbeto/angular-svg-round-progressbar#115]
<round-progress
                max="100"
                current="20"
                color="red">
                
                <round-progress max="100"
                    current="20"
                    color="blue"  offset="30"></round-progress></round-progress>

@crisbeto  the above code doesnt display the inner nested progress bar. It works well in browser.

**Comment by crisbeto** [comment]
The Angular 2 version doesn't support nesting yet.

**Comment by JosnaPriyaPereira** [comment]
@crisbeto thanks for your reply.  I missed to add more detail here. The nested progress bars display ok in the chrome browser. The app is an ionic angular 1.5 app. While i load this app on the ios device/device simulator built using xcode the inner nested bar doesnt show up. 

Also the svg linear gradients dont show up in ios device. Is this a bug with the progrss bar or device

**Comment by crisbeto** [comment]
I think I know what the issue may be with the gradients, but first: have you tried only with solid colors in Ionic?

**Comment by JosnaPriyaPereira** [comment]
Yes, the progress bar works fine with solid colors in the device but the nesting itself doesnt work either with solid colors or linear gradients. As already mentioned Linear gradients and nesting work fine in chrome browser.

**Comment by alstonkim** [comment]
@crisbeto Could you push this change to Angular2 version?
